Fails to build against Boost 1.70 (Beta1)

Regressed by boostorg/beast@9a8e22950ff5:
```c++
lib/http/session.cpp:60:22: error: no member named 'cout' in namespace 'std'
                std::cout << "shutdown: " << ec.message() << std::endl;
                ~~~~~^
lib/http/session.cpp:155:22: error: no member named 'cerr' in namespace 'std'
                std::cerr << ec.message() << "\n";
                ~~~~~^
2 errors generated.
```

--- a/lib/http/session.cpp
+++ b/lib/http/session.cpp
@@ -1,5 +1,7 @@
 #include "mtxclient/http/session.hpp"
 
+#include <iostream>
+
 using namespace mtx::http;
 
 Session::Session(boost::asio::io_service &ios,

